webpack/src/Components/EmptyState/EmptyStateActions.js

```diff
@@ -0,0 +1,20 @@
+import { API_OPERATIONS } from 'foremanReact/redux/API/APIConstants';
+import { ADD_CONTENT, FETCHING_KEY } from './Constants';
+
+export const AddEmptyStateContent = header => ({
+  type: ADD_CONTENT,
+  payload: header,
```

Comment:
I believe the `payload` should be wrapped by an object

Reply:
not it this case, the payload is just a string, it's very simple action

Reply:
IMO we should create another example, maybe something with the empty state button click which creates an API call that passes some data to the page,

passing the header feels like something that doesn't happen much.